roles/softserve/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Install required packages
+  package:
+    state: present
+    name: "{{ item }}"
+  with_items:
+  - python-virtualenv
+  - gcc
+  - git
```

Comment:
You'll definitely always need python-devel and postgresql-devel if you're compiling psycopg2

Reply:
But why compile psycopg2, and not use the existing package ?